.github/scripts/check_integration_test_label.py

```
@@ -0,0 +1,188 @@
+### 🔒 Authorization:
+Only organization members and owners can use the `/integration-tests-ok` command.
+
+---
+*This requirement ensures production stability by verifying integration tests against the latest ODH nightly build.*"""
+
+    # Check if instruction comment already exists
+    comments = pull_request.get_issue_comments()
+    for comment in comments:
+        if ("Integration Test Verification Required" in comment.body and
+            comment.user.type == "Bot"):
+            print("ℹ️ Instruction comment already exists")
```

Ruff (0.14.4)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)